suse_openstack_cloud

```diff
@@ -359,6 +359,9 @@ for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer \
         rabbitmq)
             pconf_files /etc/rabbitmq/enabled_plugins
             ;;
+        neutron)
+            pconf_files /var/lib/neutron/ha_confs/
```

Comment:
`pconf_files` expects files as arguments, not directories.  I think you want `find_and_pconf_files`.  Please mark as untested if not tested yet ;-)

Reply:
Fixed (at least the `find_and_pconf_files` part). Still needs testing (so I keep the WIP label for now)